Makefile

```diff
@@ -1,7 +1,17 @@
-REGISTRY       ?= quay.io
-ORG            ?= kubernetes_incubator
-TAG            ?= $(shell git describe --always --dirty)
-IMAGE          ?= $(REGISTRY)/$(ORG)/node-feature-discovery-operator:$(TAG)
+IMAGE_BUILD_CMD ?= docker build
+IMAGE_BUILD_EXTRA_OPTS ?=
+IMAGE_PUSH_CMD ?= docker push
```

Comment:
This is unused(?) You might want to add a `push` or `push-images` target using this as you do `make push` later in `push-image.sh`

Reply:
good catch, done!